docker/terrat/service/terrat

```diff
@@ -11,18 +11,6 @@ GITHUB_API_BASE_URL = 'GITHUB_API_BASE_URL'
 REQUIRED_ENV_KEYS = []
 
 def setup_environment():
-    if os.getenv('CUSTOM_CA_CERT'):
```

Comment:
@orbitz I don't remember what's going on here.... My assumption is that we can't run this as non-root but do we need this functionality?